Multisig/UI/Transaction/InitiateTransaction/ReviewSafeTransactionViewController/ReviewSafeTransactionViewController.swift

```diff
@@ -324,6 +323,11 @@ class ReviewSafeTransactionViewController: UIViewController {
         presentModal(ViewControllerFactory.modal(viewController: ribbon))
     }
 
+    // Please override in subclasses
+    func getTrackingEvent() -> TrackingEvent {
```

Comment:
it would be better to extract basic functionality to an abstract class so that overriding this method is mandatory. Having .assetsTransferAdvancedParams as default value is not intuitive when reading this code.

Reply:
@elgatovital Swift does not support abstract classes. We could introduce a protocol. Is it worth it?

Reply:
no it's not worth it for the protocol. I was saying that instead of a method we could use a property

Reply:
Would making it a property increase the visibility (Make it more intuitive) ?

Reply:
See: 0dd2b77ee0dfe6b65578b47272a5da8cd4a66eeb